seedfarmer/cli_groups/_taint_group.py

```diff
@@ -0,0 +1,167 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License").
+#    You may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import logging
+from typing import List, Optional
+
+import click
+
+import seedfarmer.mgmt.deploy_utils as du
+import seedfarmer.mgmt.module_info as mi
+from seedfarmer import DEBUG_LOGGING_FORMAT, config, enable_debug
+from seedfarmer.output_utils import print_bolded
+from seedfarmer.services.session_manager import ISessionManager, SessionManager
+from seedfarmer.utils import load_dotenv_files
+
+_logger: logging.Logger = logging.getLogger(__name__)
+
+
+def _load_project() -> str:
+    try:
+        _logger.info("No --project provided, attempting load from seedfarmer.yaml")
+        return config.PROJECT
+    except FileNotFoundError:
+        print_bolded("Unable to determine project to bootstrap, one of --project or a seedfarmer.yaml is required")
+        raise click.ClickException("Failed to determine project identifier")
+
+
+def _error_messaging(deployment: str, group: Optional[str] = None, module: Optional[str] = None) -> None:
+    if group and module:
+        print(f"No module info found for {deployment}-{group}-{module}")
+        print_bolded(f"To see all deployed modules in {deployment}, run seedfarmer list modules -d {deployment}")
+    else:
+        print(f"No module info found for {deployment}")
+    print_bolded("To see all deployments, run seedfarmer list deployments")
+
+
+@click.group(name="taint", help="Top Level command to support adding a taint to a deployed module")
+def taint() -> None:
+    """Taint module"""
+    pass
+
+
+@taint.command(
+    name="module",
+    help="""This command will mark a module as needing
+        redeploy of a module on the next deployment.
+        Do not use this unless you are sure of the ramifications!
```

Comment:
This sounds scary -- is there any guidance on when this should/shouldn't be used?